### PR TITLE
Add a blog for e2e develop experience

### PR DESCRIPTION
**Pull request by masayag**
Signed-off-by: Moti Asayag <masayag@redhat.com>

rh-pre-commit.version: 2.2.0
rh-pre-commit.check-secrets: ENABLED

### CODE REVIEW

File: content/blog/developer-experience-detailed.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+---
+date: 2024-03-15
+title: 'Serverless Workflows: an Automated Developer Experience Step-by-Step'
+---
+In this blog, we'll guide you through the journey from a software template to bootstrapping the workflow development, building, packaging, releasing, and deploying it on a cluster. If you need a high-level explanation or want to dive into the architecture of the solution, check out our previous blog. You can also watch a detailed demonstration of the content covered in this post in this [recording](https://www.youtube.com/watch?v=G6wnRHjvhv0).
```
Comment (y-first) [inline content/blog/developer-experience-detailed.md:5]: Should there be a link here to the previous blog?

**Comment by y-first** [comment]
Note that after the recent changes to the assessment/infrastructure workflow, the screenshots of the input parameters for the template are no longer accurate.

File: content/blog/developer-experience-detailed.md (hunk 1 above)
Comment (gciavarrini) [reply content/blog/developer-experience-detailed.md:5]: +1 for linking both the blog post and the recording.

**Review by gciavarrini — COMMENTED** [review]
As @y-first suggested, please update the screenshots. But overall it's clear and effective :)

File: content/blog/developer-experience-detailed.md (hunk 1 above)
Comment (masayag) [reply content/blog/developer-experience-detailed.md]: added.

**Comment by masayag** [comment]
@gciavarrini @y-first all comments were addressed. Pls have a look.

**Review by gciavarrini — APPROVED** [review]
lgtm

**Pull request merged**